.env.base

```diff
@@ -115,3 +115,6 @@ REACT_APP_WALLET_CONNECT_RELAY_URL=wss://relay.walletconnect.com
 
 # zerion
 REACT_APP_ZERION_API_KEY=emtfcHJvZF8xMjE3ZGRmYjhlZDY0ZGEzYmE2MjFiODg2NjQ5MTM4Yzo=
+
+# zapper
+REACT_APP_ZAPPER_API_KEY=c1b0bac5-bfc4-44f2-9fee-ed82679f76b7
```

Comment:
is this a prod key or dev key for now?

Reply:
That's effectively prod, the only limit is the 10k monthly requests, from there on we can "top it up" with points - doesn't make any localhost/prod discrimination e.g CORS